### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by unguiculus**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,29 +3,23 @@ module github.com/helm/chart-releaser
...
-	helm.sh/helm/v3 v3.4.0
-	rsc.io/letsencrypt v0.0.3 // indirect; indirect)
+	helm.sh/helm/v3 v3.5.2
 )
 
-exclude (
-	github.com/Azure/go-autorest v0.9.0
-	github.com/Azure/go-autorest v12.0.0+incompatible
+replace (
+	github.com/docker/distribution => github.com/docker/distribution v0.0.0-20191216044856-a8371794149d
```
Comment (davidkarlsen) [inline go.mod:23]: Out of interest - why is this needed? It doesn´t rewrite?

**Review by davidkarlsen — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (unguiculus) [reply go.mod:23]: I had to add that. That's due to the Helm dependency.

See https://github.com/helm/helm/blob/master/go.mod#L51-L54

Comment (unguiculus) [reply go.mod:23]: Otherwise, you get this error:

```go: helm.sh/helm/v3@v3.5.2 requires
	github.com/deislabs/oras@v0.10.0 requires
	github.com/docker/distribution@v0.0.0-00010101000000-000000000000: invalid version: unknown revision 000000000000```
